HouseRules_Essentials/HouseRules_Essentials.csproj

```diff
@@ -97,6 +97,9 @@
       <HintPath>$(SteamDemeoDir)\MelonLoader\MelonLoader.dll</HintPath>
     </Reference>
     <Reference Include="System" />
+    <Reference Include="PhotonUnityNetworking">
+      <HintPath>$(SteamDemeoDir)\demeo_Data\Managed\PhotonUnityNetworking.dll</HintPath>
+    </Reference>
```

Comment:
Turns out that it is because when making the following call
```cs
Traverse.Create(gameContext.gameStateMachine).Field<LevelSequence>("levelSequence").Value;
```

`gameStateMachine` (from `GameStateMachine.cs`) contains references to objects from this library.